### PR TITLE
Replace offensive term

### PR DESCRIPTION
**Pull request by mikelolasagasti**

### CODE REVIEW

**Review by hpatro — APPROVED** [review]
This did come up in the past in one of the internal Amazon review 😅. Thanks for the cleanup.

**Review by zuiderkwast — APPROVED** [review]
Thx. I don't know if it's offensive, but if some people think it is, then we better change it.

**Comment by AaronDewes** [comment]
This modifies 3rd party code for this purpose (`deps/linenoise/linenoise.c`). Shouldn't such a change be done upstream at https://github.com/antirez/linenoise?

**Comment by zuiderkwast** [comment]
No, that upstream not updated for many years. It's not really a dep.

**Comment by shogo82148** [comment]
Several changes have already been made, and it is now different from the upstream.

- https://github.com/valkey-io/valkey/commit/adbb534f034f94453c619b106029500823f34cf6
- https://github.com/valkey-io/valkey/commit/bbf364a442b463f3cf0d310ad052727de68a493a

How about creating a new fork? Otherwise, how about moving it from the `deps` directory?

**Comment by zuiderkwast** [comment]
> Several changes have already been made, and it is now different from the upstream.
> 
>     * [adbb534](https://github.com/valkey-io/valkey/commit/adbb534f034f94453c619b106029500823f34cf6)
> 
>     * [bbf364a](https://github.com/valkey-io/valkey/commit/bbf364a442b463f3cf0d310ad052727de68a493a)
> 
> 
> How about creating a new fork? Otherwise, how about moving it from the `deps` directory?

In theory you're right. Personnaly I don't want to prioritize maintaining a fork though. If we move it out of deps, it can be tempting to start using other include files. Currently it's independent of all other files in the project, which is kind of nice.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Top level makefile, the real shit is at src/Makefile
```
Comment (SoulPancake) [inline Makefile:1]: I find it hilarious that it was there all this time :p

**Pull request merged**